--- winexe/service.c.original	2010-09-08 12:57:03.000000000 +0200
+++ winexe/service.c	2010-09-08 12:57:36.000000000 +0200
@@ -15,7 +15,7 @@
 #define SERVICE_DEMAND_START (0x00000003)
 #define SERVICE_ERROR_NORMAL (0x00000001)
 #define SERVICE_CONTROL_STOP (0x00000001)
-#define NT_STATUS_SERVICE_DOES_NOT_EXIST NT_STATUS(0xc0000424)
+#define NT_STATUS_SERVICE_DOES_NOT_EXIST NT_STATUS(0xc00000424)
 #include "system/filesys.h"
 #include "libcli/libcli.h"
 #include "libcli/smb_composite/smb_composite.h"

